ecl/eclcc/eclcc.cpp

```diff
@@ -71,8 +71,8 @@
 
 //=========================================================================================
 
-//The following flag could be used not free items to speed up closedown
-static bool optDebugMemLeak = false;
+//The following flag is used to speed up closedown by not freeing items
```

Comment:
This logic needs to be inverted, e.g. "...by not freeing items when set to false". Normally, the description of a boolean flag refers to the true state, does it not?
or optReleaseAllMemory -> optDontReleaseAllMemory